.gitignore

```diff
@@ -53,4 +53,11 @@ docker-compose.local*.yml
 #java
 java/.classpath
 java/.project
-java/.settings/
\ No newline at end of file
+java/.settings/
```

Comment:
will get it straightened out

Reply:
you can make .gitignore files in each language directory instead of trying to cram it all in this one top level .gitignore too, that may help the conflicts here